### ISSUE TITLE
Setting custom  headers in a service subscription request

### ISSUE THREAD

**Issue by ali-bugdayci** [gautamsi/ews-javascript-api#339]
Hi,

TL;DR : How to set custom headers in a service request?

I am trying to subscribe to multiple users with a single service:

https://docs.microsoft.com/en-us/exchange/client-developer/exchange-web-services/how-to-maintain-affinity-between-group-of-subscriptions-and-mailbox-server

In order to do so, I need to set some Header values from the subscription to again subscribe with the same service. The headers that I need to set are:

- Include the X-AnchorMailbox header with a value set to the SMTP address of the anchor mailbox user for the group.

- Include the X-PreferServerAffinity header with a value set to true.

- Include the X-BackEndOverrideCookie that was returned in the anchor mailbox user's subscription response.

So after each subscription promise resolved, I need to set these and re-call it until all the members are subscribed.

**Comment by ali-bugdayci** [comment]
By the way, I am using NTLM authentication:

```
var ntlmXHRApi = new xhr.XhrApi({ rejectUnauthorized: false, gzip:true})
	.useNtlmAuthentication(config.credentials.userName, config.credentials.password);
ews.ConfigurationApi.ConfigureXHR(ntlmXHRApi);
ews.ConfigurationApi.SetXHROptions({rejectUnauthorized : false});
```

**Comment by gautamsi** [comment]
this is in README

```js
ews.<exchange service instance>.HttpHeaders.Add("header", "value");
```

**Comment by ali-bugdayci** [comment]
Thanks. missed it
